### ISSUE TITLE
Warning swallowed by DSL

### ISSUE THREAD

**Issue by Neverbirth** [DoclerLabs/hexMachina#288]
Passing some normal value to some element that expects an `Observable` produces a warning, but when doing this inside a Flow file the only information output is:
```
C:\sdk\haxe-3.4.7\std/haxe/macro/MacroStringTools.hx:86: characters 82-87 : Warning : Implicit conversion from constants to observables is deprecated and will soon be removed. See https://github.com/haxetink/tink_state/issues/9
```
Nothing else, so finding where is the problem can become a not so nice quest.

**Comment by FrancisBourre** [comment]
Could you share the flow code please?

**Comment by Neverbirth** [comment]
I guess that any warning thrown through macros would work, but having tink_state 0.7.0 it should be enough to do

```
import tink.state.Observable;
class Test {
   public function new(o:Observable<String>) {
        trace(o.value);
   }
}
```

```
plainString = "NotObservable!";
object = new Test(plainString);
```

**Comment by gene-pavlovsky** [comment]
Note: Latest tink_state update (already in master) had removed the implicit conversion to observables.
Therefore, the warning is now an error, that contains a bit more information (e.g. if you try to call a function that expects an observable with a plain value, you will get the name of the function param and the type of the observable in the error message).

**Comment by FrancisBourre** [comment]
Just tested this one, it's fixed with [1.0.0-alpha.4](https://github.com/DoclerLabs/hexMachina/releases/tag/1.0.0-alpha.4)
